### ISSUE TITLE
Cannot find -lfreetype

### ISSUE THREAD

**Issue by mortalisk** [PistonDevelopers/piston_window#122]
I can't get piston examples working with the GCC toolchain on windows. I've followed the install freetype guide, but it always ends with "cannot find -lfreetype".

**Comment by SonOfFlyingPig** [comment]
I had the same problem a couple days ago. I did manage to solve it, but unfortunately didn't document my process. I'm assuming the guide you're referring to is this: https://github.com/PistonDevelopers/freetype-sys#for-windows-users .

You might also need to explicitly add a gcc installation with:

`pacman -S mingw-w64-i686-gcc`

After that, I remember I had to manually copy the freetype libraries (libfreetype.a and libfreetype.dll.a) to the lib directory being used by cargo. On my system it was:

`C:\Program Files (x86)\Rust stable GNU 1.7\lib\rustlib\i686-pc-windows-gnu\lib`

I had to use the 32-bit versions because I got some complaint about 64-bit versions not being compatible with one of the packages.
